DNN Platform/DotNetNuke.Web/Common/DotNetNukeHttpApplication.cs

```diff
@@ -115,7 +115,7 @@ private void Application_Start(object sender, EventArgs eventArgs)
             DotNetNukeShutdownOverload.InitializeFcnSettings();
 
             // register the assembly-lookup to correct the breaking rename in DNN 9.2
-            DotNetNuke.Services.Zip.SharpZipLibRedirect.RegisterSharpZipLibRedirect();
+           // DotNetNuke.Services.Zip.SharpZipLibRedirect.RegisterSharpZipLibRedirect();
```

Comment:
Is this change supposed to be part of this PR?

Reply:
No, good catch! I'll undo that. I thought I reset things before but I guess I must of started from a different branch. Oppps :(